elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -339,6 +367,16 @@
         },
 
         /**
+         * Returns the type information of the serialized Dataframe
+         * @param df - the serialized DataFrame
+         * @return the types array
+         * @private
+         */
+        _columnTypes: function(df){
```

Comment:
So documentation is now part of the source. To update the DataFrame documentation edit [this](https://github.com/jupyter-incubator/declarativewidgets/blob/master/etc/docs/site/usage-docs/Using-DataFrames.md).